internal/quickstart/toggle_flag.go

```diff
@@ -13,6 +14,11 @@ import (
 	"ldcli/internal/flags"
 )
 
+const (
+	resetThrottleCountThreshold = 1
```

Comment:
This is a low threshold because the messages are async and a few will always come in while `resetThrottleCount` is incremeting.